doc/source/drivers/raster/stacit.rst

```diff
@@ -10,30 +10,28 @@ STACIT - Spatio-Temporal Asset Catalog Items
 
 .. built_in_by_default::
 
-This driver supports opening JSON files, or usually the result of a remote query,
-that are the ``items`` link of a
-`STAC Collection <https://github.com/radiantearth/stac-api-spec/blob/master/stac-spec/collection-spec/collection-spec.md>`_,
-and whose items also implement the
-`Projection Extension Specification <https://github.com/stac-extensions/projection/>`_.
+This driver supports opening STAC API ItemCollections, with the input usually being a `STAC API search query <https://github.com/radiantearth/stac-api-spec/tree/main/item-search>`_ or the results saved as a JSON file. Items in the response must include projection information following the `Projection Extension Specification <https://github.com/stac-extensions/projection/>`_.
 It builds a virtual mosaic from the items.
 
 A STACIT dataset which has no subdatasets is actually a :ref:`raster.vrt` dataset.
 Thus, translating it into VRT will result in a VRT file that directly references the items.
 
+Note that `STAC API ItemCollections <https://github.com/radiantearth/stac-api-spec/blob/main/fragments/itemcollection/README.md>`_ are not the same as `STAC API Collections <https://github.com/radiantearth/stac-api-spec/blob/master/stac-spec/collection-spec/collection-spec.md>`_ or `STAC Collections <https://github.com/radiantearth/stac-spec/tree/master/collection-spec>`_. STAC API ItemCollections are GeoJSON FeatureCollections enhanced with STAC entities.
```

Comment:
True, they are ultimately the same thing and could be referenced once. I included both since the STAC spec and STAC API spec live in two different places.

Reply:
I prefer to keep only one reference, to avoid more confusion. (thanks for the clarification by the way. I lived with the idea that the result of STAC API request was indeed a STAC Collection)

Reply:
It would be useful to support both types of STAC Collection as well as regular STAC Items. I think I can source some funding to expand the functionality if you are available.

Reply:
> It would be useful to support both types of STAC Collection as well as regular STAC Items

that items implement the projection extension seems to be a requirement though, otherwise the driver would have to open each item to figure out its georeferencing, which could be a performance killer for large collections

> if you are available.

yes